Src/DBProj/DBProj.sqlproj

```diff
@@ -7,7 +7,7 @@
     <SchemaVersion>2.0</SchemaVersion>
     <ProjectVersion>4.1</ProjectVersion>
     <ProjectGuid>{21f3e728-b665-46c5-bd3f-890e26bd5aaa}</ProjectGuid>
-    <DSP>Microsoft.Data.Tools.Schema.Sql.Sql120DatabaseSchemaProvider</DSP>
+    <DSP>Microsoft.Data.Tools.Schema.Sql.Sql110DatabaseSchemaProvider</DSP>
```

Comment:
This can effect other users. If you are making some change for your use, always revert that and it should not be part of commit.